docusaurus/docs-internals/editor-controls/data-controls/select.md

```diff
@@ -0,0 +1,340 @@
+      title: "None",
+    },
+    {
+      value: "thin",
+      title: "Thin",
+    },
+    {
+      value: "heavy",
+      title: "Heavy",
+    }
+  ]
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the first `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" }
+      ]
+    },
+    {
+      id: "navIcon",
+      type: "select",
+      disabled: videoType === "custom",
+      choices: [
+        {
+          value: "none",
+          title: "None",
+        },
+        {
+          value: "thin",
+          title: "Thin",
+        },
+        {
+          value: "heavy",
+          title: "Heavy",
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Helper examples
+
+The helper object contains a content property with the value `"Helper"`, which will be displayed as additional guidance or information for the user.
+
+```js
+{
+  id: "size",
+  type: "radioGroup",
+  choices: [
+    {
+      value: "none",
+      title: "None",
+    },
+    {
+      value: "thin",
+      title: "Thin",
+    },
+    {
+      value: "heavy",
+      title: "Heavy",
+    }
+  ],
+  helper: {
+    content: "Helper"
+  }
+}
+```
+
+When the helper object contains a position property with the value `"top-start"`, indicating that the helper text will be displayed at the top start of icon.
+
+```js
+{
+  id: "size",
+  type: "radioGroup",
+  choices: [
+    {
+      value: "none",
+      title: "None",
+    },
+    {
+      value: "thin",
+      title: "Thin",
+    },
+    {
+      value: "heavy",
+      title: "Heavy",
+    }
+  ],
+  helper: {
+    content: "Helper",
+    position: "top-start"
+  }
+}
+```
+
+#### ChoicesAsync example
+
+`getChoices`: This function returns a promise with choices.
+
+```js
+const getChoices = async (abortSignal: AbortSignal): Promise<Choice[]> => {
+  const response = await fetch(URL, { signal: abortSignal });
+
+  return response.json();
+};
+
+{
+  id: "navIcon",
+  type: "select",
+  choices: {
+    load: getChoices,
+    emptyLoad: {
+      title: "There are no choices"
+    }
+  }
+}
+```
+
+#### Choice with icon example
+
+```js
+{
+  id: "navIcon",
+  type: "select",
+  choices: [
+    { value: "solid", icon: { name: "nc-solid" }, title: "solid" },
+    { value: "dashed", icon: { name: "nc-dashed" }, title: "dashed" },
+    { value: "dotted", icon: { name: "nc-dotted" }, title: "dotted" }
+  ]
+}
+```
+
+#### onLoad example
+
+```js
+{
+  id: "navIcon",
+  type: "select",
+  choices: {
+    load: getChoices,
+    emptyLoad: {
+      title: "There are no choices"
+    }
+  },
+  onLoad: () => { console.warn("Choices Loaded") },
```

Comment:
console.warn -> console.log

because warn is for warning, but our `onLoad` means that choices was loaded succesfully